.github/workflows/tests.yml

```diff
@@ -477,6 +477,38 @@ jobs:
           installTics: true
           calc: ALL
 
+  trivy-repo:
```

Comment:
Lets move this into a separate workflow called security.yml, as from what i can see here its not tied to a specific LXD build and is just parsing the source code (go mod) right?

Reply:
Yes, you are correct. Will do!